RecoMET/METFilters/python/metFilters_cff.py

```diff
@@ -73,10 +79,10 @@
    primaryVertexFilter*
 #   HBHENoiseIsoFilter*
 #   HcalStripHaloFilter *
-   CSCTightHaloFilter *
+#   CSCTightHaloFilter *
 #   hcalLaserEventFilter *
 #  CSCTightHaloTrkMuUnvetoFilter *
-# CSCTightHalo2015Filter *
+   CSCTightHalo2016Filter *
```

Comment:
is this filter also valid for Run1 and 2015 data (re)processing?
If it is, the 2016 will be confusing.